cats.json

```diff
@@ -1,169 +1,665 @@
+  "########################/ /######\\ \\###########/ /#######################",
+  "| | | | | | | | | | | | \\/| | | | \\/| | | | | |\\/ | | | | | | | | | | | |",
+  "|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|"
+  ],
+  "tiger": [
+  "                        __,,,,_",
+  "       _ __..-;''`--/'/ /.',-`-.",
+  "   (`/' ` |  \\ \\ \\\\ / / / / .-'/`,_",
+  "  /'`\\ \\   |  \\ | \\| // // / -.,/_,'-,",
+  " /<7' ;  \\ \\  | ; ||/ /| | \\/    |`-/,/-.,_,/')",
+  "/  _.-, `,-\\,__|  _-| / \\ \\/|_/  |    '-/.;.\\'",
+  "`-`  f/ ;      / __/ \\__ `/ |__/ |",
+  "     `-'      |  -| =|\\_  \\  |-' |",
+  "           __/   /_..-' `  ),'  //",
+  "       fL ((__.-'((___..-'' \\__.'"
+  ],
+  "box": [
+  "  ,-.       _,---._ __  / \\",
+  " /  )    .-'       `./ /   \\",
+  "(  (   ,'            `/    /|",
+  " \\  `-\"             \\'\\   / |",
+  "  `.              ,  \\ \\ /  |",
+  "   /`.          ,'-`----Y   |",
+  "  (            ;        |   '",
+  "  |  ,-.    ,-'         |  /",
+  "  |  | (   |        hjw | /",
+  "  )  |  \\  `.___________|/",
+  "  `--'   `--'"
+  ],
+  "couple": [
+  "    |\\_                \\|\\||             ",
+  "  -' | `.             -- ||||/       ",
+  " /7      `-._        /7   |||||/            ",
+  "/            `-.____/    |||||||/`-.____________",
+  "\\-'_                \\-' |||||||||               `-._",
+  " -- `-.              -/||||||||\\                `` -`.",
+  "       |\\              /||||||\\             \\_  |   `\\\\",
+  "       | \\  \\_______...-//|||\\|________...---'\\  \\    \\\\",
+  "       |  \\  \\            ||  |  \\ ``-.__--. | \\  |    ``-.__--.",
+  "       |  |\\  \\          / |  |\\  \\   ``---'/ / | |       ``---'",
+  "     _/  / _|  )      __/_/  / _|  )     __/ / _| |",
+  "    /,__/ /,__/      /,_/,__/_/,__/     /,__/ /,__/          tbk"
+  ],
+  "tails": [
+  "                        _",
+  "                       | \\",
+  "                       | |",
+  "                       | |",
+  "  |\\                   | |",
+  " /, ~\\                / /",
+  "X     `-.....-------./ /",
+  " ~-. ~  ~              |",
+  "    \\             /    |",
+  "     \\  /_     ___\\   /",
+  "     | /\\ ~~~~~   \\ |",
+  "     | | \\        || |",
+  "     | |\\ \\       || )",
+  "    (_/ (_/      ((_/"
+  ],
+  "bug": [
+  "       _                        ",
+  "       \\`*-.                    ",
+  "        )  _`-.                 ",
+  "       .  : `. .                ",
+  "       : _   '  \\               ",
+  "       ; *` _.   `*-._          ",
+  "       `-.-'          `-.       ",
+  "         ;       `       `.     ",
+  "         :.       .        \\    ",
+  "         . \\  .   :   .-'   .   ",
+  "         '  `+.;  ;  '      :   ",
+  "         :  '  |    ;       ;-. ",
+  "         ; '   : :`-:     _.`* ;",
+  "[bug] .*' /  .*' ; .*`- +'  `*' ",
+  "      `*-*   `*-*  `*-*'"
```

Comment:
Maybe remove the bug and left the cat alone?

Reply:
lol.